src/Helpers/DistanceUtility.kt

```diff
@@ -0,0 +1,15 @@
+import kotlin.math.pow
+import kotlin.math.sqrt
+
+internal class DistanceUtility {
+
+    fun calculateDistance(from: Coordinates, to: Coordinates): Double {
```

Comment:
In the context of OOP it's a good idea since you can break up tasks into smaller parts. This way if you have a function that is used internally by the class (to serve another function) then you don't want other classes to have access to it (or to use it). An example of this could be a function that changes the value of an object inside the class, this function can be called inside the class by other functions (as an effect of some other event), but you don't want other classes to access it directly.